scss/6-components/_page_review.scss

```diff
@@ -1,6 +1,6 @@
-/** @define page-review */
-.cads-page-review {
-  @include cads-paragraph-s();
+.cads-page-review,
+.cads-prose .cads-page-review {
```

Comment:
With the change of scope in citizensadvice/public-website#458 we can likely avoid the need for this override?